### ISSUE TITLE
Investigate getting rid of external dependencies in junit-console

### ISSUE THREAD

**Issue by marcphilipp** [junit-team/junit-lambda#37]
To be usable on the command-line, it would be nice if junit-console were a jar that includes all of its dependencies (fat-jar, uber-jar, ...).

**Comment by marcphilipp** [comment]
I've discarded the fat-jar idea for now. I tried using the [shadow Gradle plugin](https://github.com/johnrengelman/shadow) to relocate jopt-simple. However, jopt-simple uses `ResourceBundles` that are loaded from `*.properties` files where keys are prefixed with class names. Those would need to be transformed to reflect the relocation as well.

Instead I've used Gradle's application plugin to publish a ZIP and TAR archives that contain shell/batch script and all of the dependencies.

The content looks like this:
![bildschirmfoto 2015-12-04 um 13 21 55](https://cloud.githubusercontent.com/assets/214207/11589677/99ff8d0e-9a8a-11e5-91bb-7a6faf55bdff.png)

---

In addition, I think we should add a command-line option to `ConsoleRunner` that would allow to specify additional classpath entries and use a custom `URLClassLoader`.
